pages/speaking/recording.vue

```diff
@@ -55,4 +78,17 @@ export default {
 }
 </script>
 
-<style scoped></style>
+<style lang="postcss" scpoed>
+.recording__intro {
+    @apply text-[18px];
+    @media (min-width: 1024px) {
+        @apply text-base;
+    }
+}
+.recording__two-col-wrapper {
```

Comment:
After creating a new PR, I think I will rewrite this part later. Like tutorial__policy in: https://github.com/pycontw/pycontw-frontend/pull/359/commits/fcb749564c66b6a7afb7904c7d8b2f47f2b66a85

Reply:
fixed: [ab21f13](https://github.com/pycontw/pycontw-frontend/pull/358/commits/ab21f1349df3ff0e591d7dbbe6a151cab53fff7e)